packages/cli/src/utils/api.js

```diff
@@ -68,6 +68,7 @@ const callAPI = async (
     method: options.method || 'GET',
     body: options.body ? JSON.stringify(options.body) : null,
     headers: {
+      ...options.extraHeaders, // extra headers first so they don't override defaults
```

Comment:
Looking for thoughts/feedback on this approach! I was surprised that I didn't find a way to override request headers. It's easier to use headers in this case due to how the API uses the entire request body.

Reply:
This makes sense to me!